faqs/emergency-shutdown.md

```diff
@@ -0,0 +1,130 @@
+
+## What happens to MakerDAO after an Emergency Shutdown?
+
+After Emergency Shutdown is triggered, the collateral redemption process takes place. In the meantime, it is up to anyone whether they want to redeploy the system or not, and how they would deploy it.
+
+## Who can redeploy the system?
+
+Anyone can redeploy the system since it is open source software.
+
+## What if multiple redeployments happen, who decides which one is legitimate?
+
+Depending on the details of each redeployment, the market, along with various system participants will have to choose the one with the most appropriate changes. Some factors include:
+
+-   No unwarranted code changes
+
+-   Changes in the distribution of the MKR token
+
+-   Changes in the Risk Parameters
+
+## Who decides how the system should be redeployed?
+
+Since it is open source software, anyone can decide. Ideally, the parameters of redeployment should depend on the reason for the Emergency Shutdown, and should not be altered unilaterally and arbitrarily. Here is a rough example of a framework for making changes on redeployment:
+
+| Reason            | Solution                                                                                                                                                                        |
+|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Governance attack | Fork out malicious MKR holders in new redistribution,  redeploy system with everything else as-is                                                                               |
+| Oracle Attack     | Fork out Oracle module for a new one with a vulnerability fix,  redeploy system with everything else as-is                                                                      |
+| Market Black Swan | Redeploy system with everything as-is, allow MKR voters to decide  how best to address this event through new or improved system mechanics  that can be added post-redeployment |
+
+## During Emergency Shutdown, will CDP owner's collateral be given away to other people?
+
+During Emergency Shutdown, collateral is immediately made available for redemption by Dai holders. This means that someone may receive a part of your original collateral. However, after the time period where CDP collateral claims are processed, any CDP owner will be able to withdraw collateral that represents the full net value of their CDP.
+
+Example:
+
+If you have a CDP with 1 ETH that's worth 300 USD, and 100 debt, you will be left with 0.6666 ETH and no debt. The Net Value of the CDP is made available for redemption, which is 300 USD value of the collateral, minus 100 USD of debt. Therefore, you will be able to claim 0.6666 ETH after the 6 hour delay period.
```

Comment:
Examples should be code fenced:

https://github.com/makerdao/community/blob/master/contributing/style-guide.md#faq-style-guide